### PR TITLE
Enterprise 0.28 release notes and minor doc changes

### PR DESCRIPTION
**Pull request by jwitz**
This PR contains release notes and minor doc updates only. Blocked by https://github.com/astronomer/docs/pull/361

Release notes are based on this thread: https://astronomer.slack.com/archives/C03070QJML4/p1644350706448959

If a change didn't have a clear user impact, I opted not to include it. This includes the CVE fixes and a few of the bug fixes. Most importantly, I'm not documenting BYO Registry capabilities yet. I think we need more time to develop docs and explain how to use this feature - currently, there's not even internal documentation related to its implementation.

### CODE REVIEW

File: enterprise/release-notes.md [hunk 1]
```diff
@@ -14,3 +14,51 @@ This document includes all release notes for Astronomer Enterprise v0.28.
...
+
+```yaml
+houston:
+  deployments:
+    namespaceFreeFormEntry: true #true|false
+    preDeploymentValidationHook: http://my-provision-hook.com/prod-us
+    preDeploymentValidationHookTimeout: 30000 # 30 sec
+```
+
+For example, you can a validation webhook can reject Deployment creation for any of the following reasons:
```
Comment (jwitz) [inline enterprise/release-notes.md:42]:
```suggestion
For example, a validation webhook can reject Deployment creation for any of the following reasons:
```

File: enterprise/release-notes.md [hunk 2]
```diff
@@ -14,3 +14,51 @@ This document includes all release notes for Astronomer Enterprise v0.28.
...
+
+- The namespace for the a Deployment is already in use.
+- The namespace for the a Deployment is in an incorrect format.
```
Comment (jwitz) [inline enterprise/release-notes.md:45]:
```suggestion
- The namespace for a Deployment is already in use.
- The namespace for a Deployment is in an incorrect format.
```

File: enterprise/release-notes.md [hunk 3]
```diff
@@ -14,3 +14,51 @@ This document includes all release notes for Astronomer Enterprise v0.28.
...
+    namespaceFreeFormEntry: true #true|false
+    preDeploymentValidationHook: http://my-provision-hook.com/prod-us
+    preDeploymentValidationHookTimeout: 30000 # 30 sec
+```
+
+For example, a validation webhook can reject Deployment creation for any of the following reasons:
+
+- The namespace for the a Deployment is already in use.
+- The namespace for the a Deployment is in an incorrect format.
+- The user creating the Deployment is unauthorized to complete this action.
```
Comment (jwitz) [inline enterprise/release-notes.md:46]:
```suggestion
- The user creating a Deployment is unauthorized to complete this action.
```

File: enterprise/release-notes.md [hunk 4]
```diff
@@ -14,3 +14,51 @@ This document includes all release notes for Astronomer Enterprise v0.28.
...
+
+### Bug Fixes
+
+- Removed root user permissions for authSidecar
+- Added AWS RDS certificates to list of trusted certificates
+- Removed support for Kubernetes 1.18
+- Fixed some confusing behavior the Git-Sync **SSH Key** field in the UI  
+- Fixed an issue where the Astronomer platform and Airflow could not communicate in environments where inter-namespace communication is disabled
+- Fixed an issue where users would frequently get 502 errors when logging in to the Astronomer UI
+- Fixed an issue users would get timeout issues when attempting to log in to an Astronomer installation on OpenShift
```
Comment (jwitz) [inline enterprise/release-notes.md:64]:
```suggestion
- Fixed an issue where users would get timeout issues when attempting to log in to an Astronomer installation on OpenShift
```

File: enterprise/release-notes.md [hunk 5]
```diff
@@ -14,3 +14,51 @@ This document includes all release notes for Astronomer Enterprise v0.28.
 Astronomer v0.28 is the latest Stable version of Astronomer Enterprise, while v0.25 remains the latest long-term support (LTS) version. To upgrade to Astronomer v0.28 from v0.26+, read [Upgrade to a Stable Version](upgrade-astronomer-stable.md). For more information about Enterprise release channels, read [Release and Lifecycle Policies](release-lifecycle-policy.md). To read release notes specifically for the Astronomer CLI, see [Astronomer CLI Release Notes](cli-release-notes.md).
 
 We're committed to testing all Astronomer Enterprise versions for scale, reliability and security on Amazon EKS, Google GKE and Azure AKS. If you have any questions or an issue to report, don't hesitate to [reach out to us](https://support.astronomer.io).
+
+## v0.28.0
+
+Release date: February 10, 2022
```
Comment (vishwas-astro) [inline enterprise/release-notes.md:20]: Hopefully tomorrow Feb 11th.

File: enterprise/release-notes.md [hunk 6]
```diff
@@ -14,3 +14,51 @@ This document includes all release notes for Astronomer Enterprise v0.28.
...
+- You can now set [Grafana environment variables](https://grafana.com/docs/grafana/latest/administration/configuration/#override-configuration-with-environment-variables) using the `grafana.extraEnvVars` setting in your `config.yaml` file.
+- Added a new **Ephemeral Storage Overwrite Gigabytes** slider to the Git Sync configuration screen. You can configure this slider to allocate more memory for syncing larger Git repos.
+- Added a new **Sync Timeout** slider to the Git Sync configuration screen. You can configure this slider to set a maximum allowed length of time for syncing a Git repo.
+
+### Bug Fixes
+
+- Removed root user permissions for authSidecar
+- Added AWS RDS certificates to list of trusted certificates
+- Removed support for Kubernetes 1.18
+- Fixed some confusing behavior the Git-Sync **SSH Key** field in the UI
```
Comment (vishwas-astro) [inline enterprise/release-notes.md:61]: with the Git-Sync  **SSH Key** field in the UI

File: enterprise/release-notes.md (hunk 5 above)
Comment (vishwas-astro) [reply enterprise/release-notes.md]: Update to Feb 14th before merging please

File: enterprise/deploy-git-sync.md [hunk 7]
```diff
@@ -60,6 +60,8 @@ Workspace editors can configure a new or existing Airflow Deployment to use a gi
     - **Rev**: The commit reference of the branch that you want to sync with your Deployment
     - **Ssh Key**: The SSH private key for your Git repository
     - **Known Hosts**: The known_hosts for your Git provider, formatted as: `<host1>,<ip1> <public_key>`
+    - **Ephemeral Storage Overwrite Gigabytes**: The storage limit for your Git repository. If your Git repo is larger than 2GB, we recommend setting this slider to your repo size + 1 Gi.
+    - **Sync Timeout**: The maximum amount of seconds allowed for a sync. We recommend increasing this value if your repo is larger than 1GB.
```
[the remark this answers is not in the document]
Comment (bote795) [reply enterprise/deploy-git-sync.md:64]: sorry, missed this. looks good

File: enterprise/release-notes.md [hunk 8]
```diff
@@ -14,3 +14,51 @@ This document includes all release notes for Astronomer Enterprise v0.28.
...
+For example, a validation webhook can reject Deployment creation for any of the following reasons:
+
+- The namespace for a Deployment is already in use.
+- The namespace for a Deployment is in an incorrect format.
+- The user creating a Deployment is unauthorized to complete this action.
+
+### Additional Improvements
+
+- Astronomer now supports `prefer` and `require` SSL modes for connecting to PGBouncer. You can set this SSL mode via the `global.ssl.mode` value in your `config.yaml` file. Note that in v0.28.0, this feature works only with AWS and Azure.
+- All metrics dashboards have been upgraded to Grafana 8.
```
Comment (vishwas-astro) [inline enterprise/release-notes.md:51]: @danielhoherd should we remove this? I dont think this is done yet, right?

Comment (danielhoherd) [reply enterprise/release-notes.md:51]: Yes. We have not migrated most of the dashboards. I took that out of 0.28.0 today.

Comment (danielhoherd) [inline enterprise/release-notes.md:51]:
```suggestion
```

File: enterprise/deploy-git-sync.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (cmarteepants) [reply enterprise/deploy-git-sync.md:64]: lgtm!

**Review by cmarteepants — COMMENTED** [review]
Looks good to me (with the changes Vishwas asked for)

File: enterprise/cli-release-notes.md [hunk 9]
```diff
@@ -7,44 +7,21 @@ description: Release notes for the Astronomer CLI.
...
-    0.27.2_a64c1a-postgres-1	running		5432
-    ```
-
-    To change the default names of these containers, run `astro config set <airflow-component>.container_name <new-component-container-name>`.
+Release date: February 15, 2022
 
 ### Additional Improvements
 
-- The Astronomer CLI can now be [installed](cli-quickstart.md) on machines with an [Apple M1 chip](https://www.apple.com/newsroom/2020/11/apple-unleashes-m1/) via both curl and Homebrew.
+- If you run `astro deployment create` when [Deployment validation webhooks](release-notes.md#apply-validation-webhooks-to-deployment-creation) are enabled, then the CLI now prompts you to specify a namespace for your new Deployment.
```
Comment (neel-astro) [inline enterprise/cli-release-notes.md:20]: I think this change is specific to JPMC and not be included in release notes, correct me if I am wrong @ajayy004

**Review by neel-astro — COMMENTED** [review]
looks fine, maybe we might want to

File: enterprise/version-compatibility-reference.md [hunk 10]
```diff
@@ -21,7 +21,7 @@ While the tables below reference the minimum compatible versions, we typically r
 | v0.25               | 1.17, 1.18, 1.19, 1.20, 1.21 | 3.6  | 0.13.5       | 9.6+     | 3.6, 3.7, 3.8, 3.9 (_requires AC 2.2.0+_) | 0.25.x         | All                  |
 | v0.26               | 1.17, 1.18, 1.19, 1.20, 1.21 | 3.6  | 0.13.5       | 9.6+     | 3.6, 3.7, 3.8, 3.9 (_requires AC 2.2.0+_) | 0.26.x         | All                  |
 | v0.27               | 1.18, 1.19, 1.20, 1.21       | 3.6  | 0.13.5       | 9.6+     | 3.6, 3.7, 3.8, 3.9 (_requires AC 2.2.0+_) | 0.27.x         | All                  |
-| v0.28               | 1.18, 1.19, 1.20, 1.21       | 3.6  | 0.13.5       | 9.6+     | 3.6, 3.7, 3.8, 3.9 (_requires AC 2.2.0+_) | 0.27.x         | All                  |
+| v0.28               | 1.19, 1.20, 1.21       | 3.6  | 0.13.5       | 9.6+     | 3.6, 3.7, 3.8, 3.9 (_requires AC 2.2.0+_) | 0.27.x         | All                  |
```
[the remark this answers is not in the document]
Comment (jwitz) [reply enterprise/version-compatibility-reference.md:24]: Sounds good. Making this change in a separate PR where I'm storing all version maintenance changes: https://github.com/astronomer/docs/pull/352

File: enterprise/cli-release-notes.md (hunk 9 above)
Comment (ajayy004) [reply enterprise/cli-release-notes.md:20]: Yes that's correct @neel-astro.

@aliotta  Please confirm, as far I know we don't want to document this in public resource.
CC: @vishwas-astro

**Pull request merged**